### DISCUSSION TITLE
Support Markdown Content in Nested Menu Index Files

### DISCUSSION THREAD

**Discussion by ghost** [colinwilson/lotusdocs#22] [Feature Requests]
**Feature Request:**

Nested menus like [Guides](https://lotusdocs.dev/docs/guides/), [Reference](https://lotusdocs.dev/docs/reference/), etc. only show the `title` and `description` from the `_index.md` markdown file.

Formatting directives like headings, text, URLs, etc. are ignored. 

Take the Guides nested menu for instance - an introduction on whats going to be covered, followed by the links to "Features", "Theme Options", and "Shortcodes" would give the reader a bigger picture and a better understanding. 

It's better than simply listing the links IMO.

**Comment by colinwilson** [comment]
I prefer (for now) to keep the nested menu pages clean and concise, and rely on the `description` from `_index.md` and the `description` from the card links on the page to guide visitors.

That being said, I agree that this should be an option for theme users. Will definitely update the list template to accommodate Markdown content.

Thanks 👍
